.github/container/Dockerfile.pax.arm64

```diff
@@ -90,30 +84,27 @@ EOT
 
 ARG BASE_IMAGE
 FROM ${BASE_IMAGE} as mealkit
-ARG REPO_PAXML
-ARG REPO_PRAXIS
-ARG REF_PAXML
-ARG REF_PRAXIS
 ARG SRC_PATH_PAXML
 ARG SRC_PATH_PRAXIS
+ARG SRC_PATH_TFTEXT
 
 COPY --from=lingvo-builder /tmp/lingvo/dist/lingvo*linux_aarch64.whl /opt/
-RUN echo "lingvo @ file://$(ls /opt/lingvo*.whl)" >> /opt/pip-tools.d/manifest.pax
+RUN echo "lingvo @ file://$(ls /opt/lingvo*.whl)" >> /opt/pip-tools.d/requirements-paxml.in
 
-COPY --from=tftext-builder /opt/tensorflow-text/tensorflow_text*.whl /opt/
-RUN echo "tensorflow-text @ file://$(ls /opt/tensorflow_text*.whl)" >> /opt/pip-tools.d/manifest.pax
+COPY --from=tftext-builder ${SRC_PATH_TFTEXT}/tensorflow_text*.whl /opt/
+RUN echo "tensorflow-text @ file://$(ls /opt/tensorflow_text*.whl)" >> /opt/pip-tools.d/requirements-paxml.in
 
 # paxml + praxis
 RUN <<"EOT" bash -ex
-echo "tensorflow==2.13.0" >> /opt/pip-tools.d/manifest.pax
-echo "tensorflow_datasets==4.9.2" >> /opt/pip-tools.d/manifest.pax
-echo "chex==0.1.7" >> /opt/pip-tools.d/manifest.pax
-echo "auditwheel" >> /opt/pip-tools.d/manifest.pax
-
-get-source.sh -f ${REPO_PAXML}  -r ${REF_PAXML}  -d ${SRC_PATH_PAXML}
-get-source.sh -f ${REPO_PRAXIS} -r ${REF_PRAXIS} -d ${SRC_PATH_PRAXIS}
-echo "-e file://${SRC_PATH_PAXML}[gpu]" >> /opt/pip-tools.d/manifest.pax
-echo "-e file://${SRC_PATH_PRAXIS}"     >> /opt/pip-tools.d/manifest.pax
+echo "tensorflow==2.13.0" >> /opt/pip-tools.d/requirements-paxml.in
+echo "tensorflow_datasets==4.9.2" >> /opt/pip-tools.d/requirements-paxml.in
+echo "chex==0.1.7" >> /opt/pip-tools.d/requirements-paxml.in
```

Comment:
Why do I see it here, but not on line 27?
I wonder, can we do something like this:

```
26: ENV EXTRA_PACKAGES="tensorflow==2.13.0 tensorflow_datasets==4.9.2 chex==0.1.7 auditwheel"
27: RUN <<"EOT" bash -exu -o pipefail
pip install ${EXTRA_PACKAGES}
get-source.sh -l tensorflow-text -m ${MANIFEST_FILE}
cd ${SRC_PATH_TFTEXT}
./oss_scripts/run_build.sh
EOT
<blah-blah-blah>
98:  RUN <<EOT bash -ex
99:  for spec_pkg in $(tr " " "\n"); do
100:    echo ${spec_pkg} >> /opt/pip-tools.d/requirements-paxml.in
101: done
```
?

Reply:
I didn't author this dockerfile so I defer to @joker-eph regarding `chex`

W.r.t `EOT` vs `EOF`, either works, but one section of the dockerfile has two nested here-docs, so you need to choose a different char sequence, otherwise bash will end the heredoc early. There's one above it using EOT, but not nested; I can change that to EOF, which is more common

Reply:
I can rename the nested one to `EOF` and `EOFINNER` to be more clear

Reply:
If I wrote it, that was likely because it was required at the time to make the container work on ARM64.
We have lot of issues with incompleteness of the ARM64 pip packages, and the order in which we install things, their incompatible requirements, etc.

Ultimately line 27 and here are trying to get to a different state (earlier is just the requirement for building PAXML I believe?).
It is also very possible that the situation evolved since (that was over the summer if I remember correctly) and remove this from here would work.